packages/map-template/.github/README.md

```diff
@@ -165,3 +165,49 @@ $ npm run deploy-to-gcloud -- BUCKET_NAME
 
 This script will only work if you've already authenticated using the `gsutil` CLI on your machine, and have the proper access rights to deploy to the specified bucket.
 
+## External customization of the Map Template
```

Comment:
I'm not entirely sure that this should go into this README file. This is about the React app, where you have full access to the MapsIndoors instance anyway.

Reply:
That's right, I've removed it from here.

Reply:
I'm still not convinced that this readme should mention anything about this 🤔 💭

Reply:
Ooops, forgot to remove that part 😄